e2etest/zt_newe2e_basic_functionality_test.go

```diff
@@ -63,4 +63,7 @@ func (s *BasicFunctionalitySuite) Scenario_SingleFileUploadDownload(svm *Scenari
 	ValidateResource[ObjectResourceManager](svm, dstObj, ResourceDefinitionObject{
 		Body: body,
 	}, true)
+
+	// Validate that the network stats were updated
+	ValidateStatsReturned(svm, stdout)
```

Comment:
just piggy backing off of the Basic Test to validate that this is being returned now.